### PR TITLE
remove notifications

### PR DESCRIPTION
**Pull request by ghost**
useless

### CODE REVIEW

**Review by Scrumplex — APPROVED** [review]

File: launcher/ui/pages/global/LauncherPage.ui [hunk 1]
```diff
@@ -38,7 +38,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```
Comment (ghost) [inline launcher/ui/pages/global/LauncherPage.ui]: I forgor to fix this

**Comment by d-513** [comment]
could we get it working actually via the new website for important alerts maybe?

**Comment by ghost** [comment]
just use RSS for that

**Comment by d-513** [comment]
RSS can get burried

**Comment by ZekeZDev** [comment]
I can work on something around it for the new website we already have rss so mabye an alert tag or something.

**Comment by ghost** [comment]
> could we get it working actually via the new website for important alerts maybe?

For the record MultiMC didn't even use this for critical security vulnerabilities, alerts can easily go on RSS

there's no point in using this

**Comment by d-513** [comment]
doesn't matter that MMC didnt use it, we can use it

**Comment by Scrumplex** [comment]
I think we can remove this. Just use news for urgent stuff.

**Review by d-513 — APPROVED** [review]

**Review by ZekeZDev — APPROVED** [review]

**Pull request merged**